scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/matcher/simplify/PrimitiveArraySimplifiers.java

```diff
@@ -44,63 +45,106 @@
  */
 public class PrimitiveArraySimplifiers implements OpCollection {
 
-	@OpHints(hints = {Simplification.FORBIDDEN})
+	@OpHints(hints = { Simplification.FORBIDDEN })
 	@OpField(names = "simplify")
 	public final Function<Byte[], ObjectArray<Number>> byteArrSimplifier =
-		b -> new ObjectArray<>(b);
-
-	@OpHints(hints = {Simplification.FORBIDDEN})
+		arr -> {
+			var oa = new ObjectArray<>(Number.class, arr.length);
+			// TODO: Why doesn't System.arraycopy work?
+			for(int i = 0; i < arr.length; i++) {
+				oa.set(i, arr[i]);
+			}
+			return oa;
+		};
+
+	@OpHints(hints = { Simplification.FORBIDDEN })
 	@OpField(names = "focus")
 	public final Function<ObjectArray<Number>, Byte[]> byteArrFocuser = o -> o
 		.stream().map(b -> b == null ? null : b.byteValue()).toArray(Byte[]::new);
```

Comment:
Why are we using `Byte[]` rather than `byte[]` here?

Reply:
Uh, no reason, we should have both though, which I have thus added!